CMakeLists.txt

```diff
@@ -538,6 +538,7 @@ set(scylla_sources
     tombstone_gc_options.cc
     tombstone_gc.cc
     reader_concurrency_semaphore.cc
+    websocket/controller.cc
```

Comment:
Commitlog should explain why we are adding this demo to scylla executable. Scylla is not a collection of demos, but adding a demo is acceptable if it introduces the base infrastructure, expanded on in a later commit.